src/include/sof/debug/debug.h

```diff
@@ -28,7 +27,19 @@
 #define DEBUG_GDB	0
 #endif
 
-#ifdef CONFIG_DEBUG
+#if CONFIG_DEBUG_LOCKS
+#define DEBUG_LOCKS	1
+#else
+#define DEBUG_LOCKS	0
+#endif
+
+#if CONFIG_DEBUG_LOCKS_VERBOSE
+#define DEBUG_LOCKS_VERBOSE	1
+#else
+#define DEBUG_LOCKS_VERBOSE	0
+#endif
+
+#if CONFIG_DEBUG
```

Comment:
Why do we need aliases for those? Maybe it's better to keep them all with CONFIG_ prefixes and remove alias that was there for GDB.

Reply:
Then I cannot simply use it in a macro, because it doesn't always exist.

Reply:
In which macro f.e.?

Reply:
DEBUG_SET_FW_READY_FLAGS

Reply:
I would prefer to not have 2 symbols representing the same thing. You can play with #ifs or create macro that converts CONFIGs to bools that can be used f.e. in ternary operators.

Reply:
@tlauda @dbaluta  I mentioned in the past that we need something like IS_ENABLED etc stuff related to kconfig (look here: https://github.com/torvalds/linux/blob/master/include/linux/kconfig.h) with something like that you would be able to just use `IS_ENABLED(DEBUG)?a:b`. However I wasn't sure how to handle this because idk how we are going to handle modules...